jupyterhub_singleuser_profiles/service.py

```diff
@@ -37,91 +39,91 @@ def __init__(self, server_url, token, namespace=None, verify_ssl=True):
     self.k8s_api_instance = client.CoreV1Api(k8s_client)
     self.os_client = DynamicClient(k8s_client)
 
-  def get_template(self, name):
-    tmplt = self.os_client.resources.get(api_version='template.openshift.io/v1', kind='Template')
-    if isinstance(tmplt, list):
-      tmplt = tmplt[0] # Openshift returns list of 2 resource types, we want template.openshift.io/v1/processedtemplates
+  def get_support_config_map (self):
+    config_map_wrapper = self.os_client.resources.get(api_version='v1', kind='ConfigMap')
+    try:
+      result = config_map_wrapper.get(name='singleuser-support-configmap')
```

Comment:
I'd not user term "support". It is more like "service resource reference", so `singleuser-service-ref-${user}` - e.g. `singleuser-service-ref-vpavlin`. Then you need to pass `user` into the `get_service_reference_config_map` as param and get a CM for the user